src/redux/sagas/motions/finalizeMotion.ts

```diff
@@ -75,9 +40,13 @@ function* finalizeMotion({
       ActionTypes.TRANSACTION_CREATED,
     );
 
+    const gasLimit = BigNumber.from(gasEstimate).lte(DEFAULT_GAS_LIMIT)
```

Comment:
Instead of estimating here, we can just take the estimate off the motion (which will be more accurate when it comes to multicall) and check it doesn't exceed the block limit